src/resources/i18n-target.html

```diff
@@ -0,0 +1,52 @@
+<template>

```

Comment:
I'd move this file into `resources/plugins/i18n/i18n-target.html`. This way we can start a proper separation if the same app is going to be used for various other plugins.

Reply:
Done